### ISSUE TITLE
Documents must have <title> element to aid in navigation

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1159]
Ensures each HTML document contains a non-empty <title> element.

```
Highlight element #sbox-iframe,html
<html class="cp-app-noscroll">
Failure Details
Document does not have a non-empty <title> element
```
